analysis/tc_tackle_testapp_public_binary.go

```diff
@@ -21,22 +21,8 @@ var TackleTestappPublicBinary = TC{
 	},
 	Binary: true,
 	Analysis: api.Analysis{
-		Effort: 15,
+		Effort: 1,
```

Comment:
These analysis results are slightly different to previous ones, however, the output looks reasonable to me. Could @jmle or @pranavgaikwad confirm?